rfc/rfc008-certificate-structure.md

```diff
@@ -33,98 +33,47 @@ When it comes to certificates, we’d like to be at the edge of usability and se
 ## 2. Terminology
 
 * **CA**: Certificate authority as defined by [RFC2459](https://tools.ietf.org/html/rfc2459).
-* **event type**: the type of entry in the document log.
+* **CPS**: Certification Practise Statement.
+* **CRL**: Certificate Revocation List. 
 * **node**: a running instance of a Nuts implementation \(a.k.a. Nuts node\).
 * **node operator**: an organization running a node. Typically a vendor that creates healthcare software for care organizations.
-* **shared network state**: the accumulation of events or document log entries that are in the right order and where every node agrees upon.
-* **truststore**: The file that contains the trusted CA's.
+* **PKIo**: Public Key Infrastructure overheid. The by the Dutch government controlled PKI structure.
+* **TSP**: Trusted Service Provider. Party that can issue PKIo certificates. 
 
 Other terminology is taken from the [Nuts Start Architecture](rfc001-nuts-start-architecture.md#nuts-start-architecture).
 
-## 3. Overview
+## 3. PKIoverheid
```

Comment:
Do we want to be this explicit or leave it up to the "network certification"?

Reply:
Good question, I think leave it up to the network, so we should describe PKIoverheid as option but make it non-normative. Then development/test networks could be compliant as well.